hack/compare_scc.sh

```diff
@@ -36,7 +36,7 @@ function dump_sccs_before(){
 }
 
 function dump_sccs_after(){
-    if [ "${CMD}" == "oc" ]; then
+    if [ "${CMD}" == "oc" ] && [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
```

Comment:
why is this skipped when KUBEVIRT_PROVIDER == external?

Reply:
without skipping this, it fails when redeployed on CRC
I didn't check deeper

Reply:
It fails because of this (on 2nd deploy and so on)

```
$ diff -I '^\s*generation:.*$' -I '^\s*resourceVersion:.*$' -I '^\s*time:.*$' _out/scc/containerized-data-importer.yaml.before _out/scc/containerized-data-importer.yaml.after
79c79,81
< users: []
---
> users:
> - system:serviceaccount:kubevirt-hyperconverged:cdi-sa
> - system:serviceaccount:kubevirt-hyperconverged:cdi-cronjob

```

Reply:
thanks, let's eventually address it in a follow up

Reply:
https://github.com/kubevirt/hyperconverged-cluster-operator/issues/2210